scripts/data_catalog.py

```
@@ -0,0 +1,161 @@
+from tabulate import tabulate
+from src.infrastructure.data_reference import ExternalDataManager
+
+
+def get_file_info(file_path: Path) -> Dict:
+    file_stats = file_path.stat()
+    size_mb = file_stats.st_size / (1024 * 1024)
+    
+    info = {
+        'file': file_path.name,
+        'size_mb': round(size_mb, 2),
+        'modified': datetime.fromtimestamp(file_stats.st_mtime).strftime('%Y-%m-%d %H:%M')
```

Ruff (0.12.2)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)